### ISSUE TITLE
[BUG] Technical tests failing while using tutorial images

### ISSUE THREAD

**Issue by Y-a-s-h-M-i-t-t-a-l** [IMDA-BTG/aiverify#173]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

I tried generating a report from Images-Pipeline with as depicted in the tutorial by IMDA. The fashion MINIST dataset and using the files given in the description link. Given the files in the arguments as specified in the documentation. Sensitive features were decided randomly from it as it was not specified in the documentation. 
Completing all steps, the test generates error overall. 

### Current Behavior

All three of the robustness, shap, fairness tests generate error. 

### Expected Behavior

All tests should be running fine and generating report. SHAP and Robustness arguments and files were exactly followed with documented tutorial, and should not be any chance of their error.

### Steps To Reproduce

![a1](https://github.com/IMDA-BTG/aiverify/assets/104529934/1e0c7c3e-838a-4189-a3d2-02f1a87dae06)
![a2](https://github.com/IMDA-BTG/aiverify/assets/104529934/d758c62b-ed35-466f-9306-ba369ce6037c)
![a3](https://github.com/IMDA-BTG/aiverify/assets/104529934/d3cfcc30-fbdf-41d9-9e9c-0f528d491d12)
![a4](https://github.com/IMDA-BTG/aiverify/assets/104529934/6b83051f-58fd-4deb-b7f0-8560a23f46f3)

### Environment

```markdown
- Operating System and Version: Windows11 10.0.22621
- AI Verify Version: No idea on how to check. (downloaded the most recent setup on 16-08-23)
- Any other relevant information (e.g. version of related package dependencies installed):

Did you build using source code or from docker file? If from source code, please provide the below information: Docker file
- Python Version:
- Node Version:
- Redis Version:
- Mongo Version: 
- Screen Resolution:
```

### Screenshots/ Code snippets

![ss1](https://github.com/IMDA-BTG/aiverify/assets/104529934/eccbc66b-a328-4c6c-afde-7e730a6e2f61)
![ss2](https://github.com/IMDA-BTG/aiverify/assets/104529934/61e9064e-6252-4bc7-ad61-b4f4edfccba0)
![ss3](https://github.com/IMDA-BTG/aiverify/assets/104529934/995dac62-cdfb-4507-81d3-017756a1d7e2)

**Comment by imda-benedictlee** [comment]
Hi @Y-a-s-h-M-i-t-t-a-l, thank you for raising this issue to our attention, and apologies for the delay in response. We have conducted some tests internally and have been able to replicate this issue on Docker on v0.9.2 release. However, we would like to highlight that we do not support image testing for FMTC and SHAP, and so it is expected to fail. For Robustness, we are currently investigating the issue in detail and will update asap. In the meantime, we suggest using the `aiverify-dev` built script as we have checked that it is working. However, do note that Robustness will take a while to complete testing.

Feel free to reach out to me or @kimeetok if you have additional queries or require further clarification.

**Comment by imda-benedictlee** [comment]
Hi @Y-a-s-h-M-i-t-t-a-l, while we are still in the midst of fixing the Dockerfile, we have tested release tag v0.9.1 and can confirm that it works. Below are the steps:

1. Git clone aiverify using the following command `git clone https://github.com/imda-btg/aiverify.git --branch=v0.9.1 --single-branch`
2. After cloning the folder, use the following command in the setup-aiverify `cd aiverify-user`
3. Modify the Dockerfile by replacing line 60 with `RUN git clone https://github.com/imda-btg/aiverify.git --branch=v0.9.1 --single-branch`
4. Save the Dockerfile
5. Following the steps [here ](https://imda-btg.github.io/aiverify/getting-started/docker-setup/)to build the Docker Image and run the Docker Container

Do let me or @kimeetok know if you continue facing issues.

**Comment by imda-benedictlee** [comment]
Below is a screenshot of the generated report:
![image](https://github.com/IMDA-BTG/aiverify/assets/118954493/ee9b8114-b0bb-43e6-bb73-6ba65fc6c758)

**Comment by imda-benedictlee** [comment]
Hi @Y-a-s-h-M-i-t-t-a-l, we have just released AI Verify v0.9.3. It should solve the issue. Do try it out and let us know if it works for you.
